CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+Contributing to IRIDA
+=====================
+
+Thank you for your interest in contributing to the Integrated Rapid Infectious Disease Analysis (IRIDA) project.  IRIDA is primarily developed by the IRIDA consortium, but the development team is happy to accept contributions from the community.  If you have changes or additions you would like to make we would love to talk about it!  Before contributing code, please first discuss the change you are planning to make with us by creating or commenting on a GitHub issue, or you can contact us by one of the other options below.
+
+
+Contacting the IRIDA Team
+-------------------------
+* To ask general or usage questions, email the IRIDA team at IRIDA-mail@sfu.ca
+* Report bugs, suggest features, or contribute code on [GitHub](http://github.com/phac-nml/irida)
+* Chat with the developers on [Gitter](https://gitter.im/irida-project/)
+
+
+Types of Contributions
+----------------------
+
+### Reporting a bug
+
+Reporting a bug is the easiest, and one of the most useful ways you can contribute to IRIDA.  IRIDA uses a [GitHub issue board] to track our issues.  If you find an issue while using IRIDA that you feel is a problem with the software, first check out our [GitHub issue board] to see if someone has already reported the issue.  If you find a similar issue, we would still like to hear about your problem.  Add a comment to the issue stating how your problem occurred.
+
+**If you find a security vulnerability, please contact the IRIDA team by email.**
+
+If you do not find a similar issue, please create a new issue with (at least) the following information:
+
+1. Where were you using IRIDA? (public IRIDA, private install, virtual machine, etc)
+2. What were you doing when the issue happened?
+3. What did you expect to happen?
+4. What actually happened?
+
+When you're done, please label the new issue as a `bug` and we will triage it as soon as we can.
+
+Note: IRIDA's GitHub issue board is for bug reports or feature requests only.  You should not use it for usage issues, problems with analyzing your data, or problems setting up IRIDA.  For issues of this nature, please contact the IRIDA team by one of the other options in the [Contacting the IRIDA Team](#contacting-the-irida-team) section.
+
+### Contributing an analysis pipeline
+
+If you have an analysis tool that you would like to build for IRIDA we'd like to hear about it!  IRIDA has a plug-in style system for analysis workflows.  This helps make it easier for tool developers to get the benefits of IRIDA's data management system and easy to use interface and incorporate their analysis workflows into IRIDA's growing suite of tools.  For an analysis pipeline to be used in IRIDA, it must be developed as a Galaxy workflow and operate on sequencing reads.  Read more about building tools for IRIDA in the [IRIDA Pipeline Development Guide](https://irida.corefacility.ca/documentation/developer/tools/pipelines/).
```

Comment:
Maybe also add a link to our example plugin (https://github.com/phac-nml/irida-plugin-example)?  Especially useful now since our docs do not mention our plugin system (at least not until the next release).

Reply:
Done in 93dcedc3a5